src/sortStudents.ts

```diff
@@ -1,16 +1,52 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function average(arr: number[]): number {
```

Comment:
```suggestion
function getAverageGrades(arr: number[]): number {
```
Let's get used to name function according to these rules:
![image](https://user-images.githubusercontent.com/96409563/212291772-6af57793-cd62-473e-8c96-5bffcd921c15.png)